src/ScreenshotWindow.vala

```diff
@@ -560,5 +569,14 @@ namespace Screenshot {
         private void cancel_clicked () {
             destroy ();
         }
+
+        // Save main window position so that this position can be used
+        // when the window reappears again
+        private void remember_window_position () {
+	    Gdk.Window window = this.get_toplevel ().get_window ();
```

Comment:
Maybe use `var` here? If you want to use the type explicitly this should be `Gdk.Window?` not `Gdk.Window` with the `?` to make it nullable.